lib/theme/dark_theme.dart

```diff
@@ -1,188 +1,189 @@
 import 'package:flutter/material.dart';
-
-import 'app_themes.dart';
 import 'colors.dart';
 
 Brightness brightness = Brightness.dark;
-VisualDensity visualDensity = defaultDensity;
-MaterialColor primarySwatch = createMaterialColor(WitnetPallet.darkShade);
-Color primaryColor = WitnetPallet.darkShade;
+MaterialColor primarySwatch =
+    createMaterialColor(WitnetPallet.opacityWitnetGreen);
+Color primaryColor = WitnetPallet.opacityWitnetGreen;
+TextSelectionThemeData textSelectionTheme =
+    TextSelectionThemeData(cursorColor: WitnetPallet.opacityWitnetGreen);
 Brightness primaryColorBrightness = brightness;
-
-TooltipThemeData tooltipTheme = TooltipThemeData(
-    height: 10,
-    padding: EdgeInsets.all(5),
-    margin: EdgeInsets.all(5),
-    verticalOffset: 0.0,
-    preferBelow: false,
-    decoration: BoxDecoration(
-      color: WitnetPallet.darkShade6,
-    ),
-    textStyle: TextStyle(color: WitnetPallet.darkShade6));
 TextTheme textTheme = TextTheme(
-  headline3: TextStyle(
-    fontFamily: 'OpenSans',
-    fontSize: 45.0,
-    fontWeight: FontWeight.w100,
-  ),
-  headline1: TextStyle(fontFamily: 'Quicksand'),
-  headline2: TextStyle(fontFamily: 'Quicksand'),
-  headline4: TextStyle(fontFamily: 'Quicksand'),
-  headline5: TextStyle(fontFamily: 'NotoSans'),
-  headline6: TextStyle(fontFamily: 'NotoSans'),
-  subtitle1: TextStyle(fontFamily: 'NotoSans'),
-  bodyText1: TextStyle(fontFamily: 'NotoSans'),
-  bodyText2: TextStyle(fontFamily: 'NotoSans'),
-  subtitle2: TextStyle(fontFamily: 'NotoSans'),
-  overline: TextStyle(fontFamily: 'NotoSans'),
-);
-
-AppBarTheme appBarTheme = AppBarTheme(
-  color: WitnetPallet.darkShade6,
-  backgroundColor: Colors.black,
-  //foregroundColor: ,
-  //elevation: ,
-  //shadowColor: ,
-  //shape: ,
-  //iconTheme: ,
-  //actionsIconTheme: ,
-  //centerTitle: ,
-//
-  //titleSpacing: ,
-  //toolbarHeight: ,
-  //toolbarTextStyle: ,
-  //systemOverlayStyle: ,
-);
-
+    headline1: TextStyle(
```

Comment:
There are some fields agnostic to the light / dark theme. Could we put them in a single file to avoid repeat them in both? Should they be in `wallet_theme`?